tests/test_bump_find_increment.py

```diff
@@ -47,17 +66,18 @@
 
 
 @pytest.mark.parametrize(
-    "messages, expected_type",
+    "messages, expected_type, regex, increment_map",
     (
-        (PATCH_INCREMENTS_CC, "PATCH"),
-        (MINOR_INCREMENTS_CC, "MINOR"),
-        (MAJOR_INCREMENTS_CC, "MAJOR"),
-        (NONE_INCREMENT_CC, None),
+        (PATCH_INCREMENTS_CC, "PATCH", defaults.bump_pattern, defaults.bump_map),
+        (MINOR_INCREMENTS_CC, "MINOR", defaults.bump_pattern, defaults.bump_map),
+        (MAJOR_INCREMENTS_CC, "MAJOR", defaults.bump_pattern, defaults.bump_map),
+        (MAJOR_INCREMENTS_REGEX_CC, "MAJOR", BUMP_PATTERN_REGEX_CC, BUMP_MAP_REGEX_CC),
+        (NONE_INCREMENT_CC, None, defaults.bump_pattern, defaults.bump_map),
     ),
 )
-def test_find_increment(messages, expected_type):
+def test_find_increment(messages, expected_type, regex, increment_map):
     commits = [GitCommit(rev="test", title=message) for message in messages]
-    increment_type = bump.find_increment(commits)
+    increment_type = bump.find_increment(commits, regex, increment_map)
```

Comment:
Seems to me that the regex from conventional commits has not been updated, as far as I see, only the tests are using the new regex. Before, by not passing `increment_map` we were using the default one.
Yon can also change `BUMP_PATTERN_REGEX_CC` to point to `defaults.bump_pattern`

`https://github.com/commitizen-tools/commitizen/blob/master/commitizen/defaults.py#L19`

Reply:
Seems fair. Changed default `bump_pattern` and `bump_map`